cobbler/items/distro.py

```diff
@@ -39,8 +39,7 @@
     ["arch", 'x86_64', 0, "Architecture", True, "", utils.get_valid_archs(), "str"],
     ["autoinstall_meta", {}, 0, "Automatic Installation Template Metadata", True, "Ex: dog=fang agent=86", 0, "dict"],
     ["boot_files", {}, 0, "TFTP Boot Files", True, "Files copied into tftpboot beyond the kernel/initrd", 0, "list"],
-    ["boot_loader", "<<inherit>>", 0, "Boot loader", True, "Network installation boot loader",
-     utils.get_supported_system_boot_loaders(), "str"],
+    ["boot_loaders", "<<inherit>>", "<<inherit>>", "Boot loaders", True, "Network installation boot loaders", 0, "list"],
```

Comment:
Upps. That was something I did during merging. Please revert.

Reply:
Rechecked. distro as well as profile and system can have a list of boot loaders.